### PR TITLE
Remove autoprefixer and add inline prefixes

### PR DESCRIPTION
**Pull request by anthonydillon**
Done
Remove autoprefixer and add inline prefixes. Simplify the npm scripts.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check the site still looks the same

### CODE REVIEW

File: static/sass/_pattern_navigation.scss [hunk 1]
```diff
@@ -426,17 +432,23 @@ $ubuntu-logo-width--mobile: $ubuntu-logo-svg-width + 2 * $sph-intra;
   }
 
   .slide-animation {
+    // sass-lint:disable no-vendor-prefixes property-sort-order
     transform: translateY(-101%);
+    -webkit-transform: translateY(-101%);
```
Comment (huwshimi) [inline static/sass/_pattern_navigation.scss]: Should be before the unprefixed transform.

File: static/sass/takeovers/_financial-services.scss [hunk 2]
```diff
@@ -64,25 +68,33 @@
...
       transform: translateX(-10%);
       width: 66%;
       z-index: 1;
+      // sass-lint:enable no-vendor-prefixes property-sort-order
     }
 
     .aubergine-cloud {
+      // sass-lint:disable no-vendor-prefixes property-sort-order
       animation: fade-left 1.25s cubic-bezier(.215, .61, .355, 1) .5s 1 forwards;
+      -webkit-animation: fade-left 1.25s cubic-bezier(.215, .61, .355, 1) .5s 1 forwards;
```
Comment (huwshimi) [inline static/sass/takeovers/_financial-services.scss]: Move above `animation`.

**Comment by barrymcgee** [comment]
@anthonydillon Do you need to remove the autoprefixer changes from this PR?

**Comment by anthonydillon** [comment]
@barrymcgee no, closing for now.

**Pull request closed without merging**